spec/stashcache.spec

```diff
@@ -65,14 +96,9 @@ exit 1
 %install
 mkdir -p %{buildroot}%{_sysconfdir}/xrootd
 make install DESTDIR=%{buildroot}
-for src in "./configs/xrootd-stashcache-origin-server.cfg.in" "./configs/xrootd-stashcache-cache-server.cfg.in"; do
-    dst=$(basename "$src" .cfg.in)
-    sed -i -e "s#@LIBDIR@#%{_libdir}#" "$src"
-    sed -e "s#@REDIRECTOR@#%{redirector_prod}#" \
-        "$src" > "%{buildroot}%{_sysconfdir}/xrootd/${dst}.cfg"
-    sed -e "s#@REDIRECTOR@#%{redirector_itb}#" \
-        "$src" > "%{buildroot}%{_sysconfdir}/xrootd/${dst}-itb.cfg"
-done
+
+# Create xrootd certificate directory
+mkdir -p %{buildroot}%{_sysconfdir}/grid-security/xrd
 
 %clean
```

Comment:
%clean is also only for RHEL5.

Reply:
as is %defattr(-,root,root) below but GitHub won't let me put a line comment there.